src/main.c

```diff
@@ -633,10 +515,8 @@ int render_client(void)
 	render_clear(&tcam);
 	if(map_enable_autorender)
 	{
-		render_cubemap((uint32_t*)NULL,
-			screen_width, screen_height, 0/4,
-			&tcam, clmap,
-			NULL, 0, '1', '0', 1.0f, 0);
+		render_cubemap(NULL, screen_width, screen_height, 0/4, &tcam,
+			       clmap, NULL, 0, '1', '0', 1.0f, 0);
```

Comment:
Current coding conventions mandate tabs. Do not use spaces to indent. If we ever switch to spaces, this would still not pass as you have tabs and spaces on the same line.